shared/constants/unlock-folders.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+
+import HiddenString from '../util/hidden-string'
+import type {TypedAction} from '../constants/types/flux'
+import type {DeviceID} from '../constants/types/flow-types'
+
+export type Device = {
+  type: 'desktop' | 'mobile',
```

Comment:
Not sure if there are other types here. `common.avdl` isn't helpful it just says this is a string.